deploy/olm/README.md

```diff
@@ -0,0 +1,6 @@
+# cert-manager operator deployment
+
+This directory cotnains all files that are used to build the [operator](https://operatorhub.io/what-is-an-operator) to deploy cert-manager.
```

Comment:
`contains`

Reply:
It'd be worth mentioning in the first sentence here exactly who we expect to be using this, e.g. not the majority of users (only those consuming through OperatorHub/OpenShift etc). We should also include a note to explain the 'status' of it, and that it is a new deployment method that we are experimenting with in order to better enable OpenShift users.